From 57905722ac36975d9d300853128c739d9d0f9169 Mon Sep 17 00:00:00 2001
From: Raj Rajasekaran <b10872@freescale.com>
Date: Fri, 16 May 2008 14:30:43 -0500
Subject: [PATCH] ENGR00073754: Build breaks if MX37-3stack keypad is disabled.

Added code to include keypad related variables only if the driver is enabled.

Signed-off-by: Raj Rajasekaran <b10872@freescale.com>
---
 arch/arm/mach-mx37/mx37_3stack.c |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/arch/arm/mach-mx37/mx37_3stack.c b/arch/arm/mach-mx37/mx37_3stack.c
index cc227a7..46473ea 100644
--- a/arch/arm/mach-mx37/mx37_3stack.c
+++ b/arch/arm/mach-mx37/mx37_3stack.c
@@ -172,13 +172,14 @@ static u16 keymap[] = {
 	KEY_RIGHT, KEY_UP, KEY_LEFTALT,
 	KEY_TAB, KEY_ESC,
 };
+#endif
 
 static struct mxc_keyp_platform_data keypad_data = {
 	.matrix = keymap,
 	.active = gpio_keypad_active,
 	.inactive = gpio_keypad_inactive,
 };
-#endif
+
 static struct i2c_board_info mxc_i2c0_board_info[] __initdata = {
 	{
 	 .driver_name = "TSC2007",
-- 
1.5.4.4

